book/src/languages.md

```diff
@@ -71,6 +71,7 @@ These configuration keys are available:
 | `text-width`          |  Maximum line length. Used for the `:reflow` command and soft-wrapping if `soft-wrap.wrap-at-text-width` is set, defaults to `editor.text-width`   |
 | `rulers`              | Overrides the `editor.rulers` config key for the language. |
 | `path-completion`     | Overrides the `editor.path-completion` config key for the language. |
+| `word-completion`     | Overrides the `editor.word-completion` config key for the language. |
```

Comment:
missing the override of `word-completion-trigger-length` (and should probably include @RoloEdits suggestion of merging under `[word-completion] enable; trigger-length;`